### ISSUE TITLE
jupyterlab/keras-tensorflow issue

### ISSUE THREAD

**Issue by Erinaceida** [plaidml/plaidml#1735]
When running the example code in jupyter lab get into strange issues, that I find really hard to reproduce. 

- At times, it works alright, either on CPU or on GPU. (very rarely)
- Sometimes it doesn't want to switch to GPU and I have to shut down the kernel or specify more details, such as `os.environ["PLAIDML_EXPERIMENTAL"] = "2"
os.environ["PLAIDML_DEVICE_IDS"] = "opencl_amd_gfx1010.0"` , but not always (shutting kernel down and restarting or giving the extra info doesn't always help).
- Sometimes after a successful run, trying to re-run it gives me this error message: `module 'tensorflow' has no attribute 'get_default_graph'`, which can be avoided when specifying an import to `import tensorflow.keras.applications as kapp`. 

- - but this makes python ignore my GPU and go straight for CPU, no matter how much I try to specify I want to use GPU

The console through which I open jupyter lab throws me this (on different iterations of what I just described), maybe it will be of use to anyone: 

`2021-03-25 21:09:54.043518: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-03-25 21:09:54.044225: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'nvcuda.dll'; dlerror: nvcuda.dll not found
2021-03-25 21:09:54.044477: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2021-03-25 21:09:54.046347: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: DESKTOP-JSFLG9N
2021-03-25 21:09:54.046473: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: DESKTOP-JSFLG9N
2021-03-25 21:09:54.046900: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-03-25 21:09:54.047284: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-03-25 21:09:55.837251: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:116] None of the MLIR optimization passes are enabled (registered 2)`

The example code I'm talking about: 

`import numpy as np
import os
import time

os.environ["KERAS_BACKEND"] = "plaidml.keras.backend"

import keras
import keras.applications as kapp
from keras.datasets import cifar10

(x_train, y_train_cats), (x_test, y_test_cats) = cifar10.load_data()
batch_size = 8
x_train = x_train[:batch_size]
x_train = np.repeat(np.repeat(x_train, 7, axis=1), 7, axis=2)
model = kapp.VGG19()
model.compile(optimizer='sgd', loss='categorical_crossentropy',
              metrics=['accuracy'])

print("Running initial batch (compiling tile program)")
y = model.predict(x=x_train, batch_size=batch_size)

# Now start the clock and run 10 batches
print("Timing inference...")
start = time.time()
for i in range(10):
    y = model.predict(x=x_train, batch_size=batch_size)
print("Ran in {} seconds".format(time.time() - start))`

Any help would be very much appreciated!
